jekyll/_cci2/deployment_integrations.md

```diff
@@ -177,8 +177,11 @@ Refer to the full example in the [2.0 Project Tutorial]( {{ site.baseurl }}/2.0/
 
 ## Google Cloud
 
-Ensure that the Google Cloud SDK is installed in your primary container so that `gcloud` and all of the necessary tools for manipulating Kubernetes resources are at your disposal inside your deployment script/commands. In the following example, if the build passes and the current branch was the master branch, CircleCI 
-runs `deploy.sh` to do the actual deployment work.
+Before deploying to Google Cloud, ensure that you have authenticated the `gcloud` command line tool. To do that, you can read the [Authenticating Google Cloud Platform]({{ site.baseurl }}/2.0/) document.
```

Comment:
this should link to  /2.0/google-auth.md

Reply:
I think this section should also link to /2.0/google-container-engine.md

Reply:
whoops good catch!